### PR TITLE
MainCameraProblem fixed plus changed shader of VR hands

### PR DESCRIPTION
**Pull request by Romeeey**
# Checklist
(Please judge for yourself which entry is valuable for your PR)

## Scene
* [x] FeatureFlags reverted to right value

# Testing
* [x] Merged _main_ into this branch and tested with the latest features
* [x] Tested with PCVR and one mobile VR device (Pico/Quest)

## Dependencies
* [x] If libphoenix-shared.dll is changed, .so is also changed

### CODE REVIEW

File: Assets/Romey/DropdownController.cs [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        if (dropdown.value == 2)
+        {
+            snapTurn.enabled = false;
+            continuousTurn.enabled = true;
+        }
+
+    }
+
+    // Update is called once per frame
+    void Update()
```
Comment (JucanAndreiDaniel) [inline Assets/Romey/DropdownController.cs:64]: Don't you already do this in LoadFont.cs?

File: Assets/Romey/DropdownController.cs [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        continuousTurn = locomotionsystem.GetComponent<ActionBasedContinuousTurnProvider>();
+
+        
+
+    }
+
+    void DropdownItemSelected(TMP_Dropdown dropdown)
+    {
+
+        if (dropdown.value == 0)
```
Comment (JucanAndreiDaniel) [inline Assets/Romey/DropdownController.cs:45]: I feel like a switch statement would be better here instead of multiple ifs

**Review by JucanAndreiDaniel — CHANGES_REQUESTED** [review]
Some major deviations from how the rest of the project is structured. Until #80 is done please have scripts loaded / objects created in PhoenixImporter.cs

And do we need now the menu things also included? If yes, please create another PR for them.

File: .gitignore [hunk 3]
```diff
@@ -68,3 +68,7 @@ Assets/UberLogger.meta
 Assets/UberLogger/
 Assets/Pickle.meta
 Assets/Pickle/
+ProjectSettings/ShaderGraphSettings.asset
+ProjectSettings/Packages/com.unity.testtools.codecoverage/Settings.json
+Assets/XR/Settings/OpenXR Package Settings.asset
+Assets/StreamingAssets/GameSettings.json
```
[the remark this answers is not in the document]
Comment (Romeeey) [reply .gitignore:74]: Everybody has his own settings. Isn't it a good idea to exclude it from commits?

[the remark this answers is not in the document]
Comment (JucanAndreiDaniel) [reply .gitignore:74]: That's why we use gamesettings.dev.json which are already excluded

[the remark this answers is not in the document]
Comment (Romeeey) [reply .gitignore:74]: Reverted in last commit

File: Assets/StreamingAssets/GameSettings.json
[the remark this answers is not in the document]
Comment (Romeeey) [reply Assets/StreamingAssets/GameSettings.json:1]: Reverted in last commits.

File: Assets/Romey/LoadFont.cs
[the remark this answers is not in the document]
Comment (Romeeey) [reply Assets/Romey/LoadFont.cs:1]: Discarded for now. I will improve it on a different branch.

File: Assets/Romey/ImageController.cs
[the remark this answers is not in the document]
Comment (Romeeey) [reply Assets/Romey/ImageController.cs:1]: Discarded for now. I will improve it on a different branch.

File: Assets/Romey/DropdownController.cs (hunk 1 above)
Comment (Romeeey) [reply Assets/Romey/DropdownController.cs]: Discarded for now. I will improve it on a different branch.

File: Assets/Romey/DropdownController.cs (hunk 2 above)
Comment (Romeeey) [reply Assets/Romey/DropdownController.cs]: Discarded for now. I will improve it on a different branch.

**Comment by Romeeey** [comment]
> UI elements were removed. Gamedev Settings file restored.

**Review by JucanAndreiDaniel — COMMENTED** [review]
Looks fine, but test the teleportation to see if it still works, on my end it didn't.
Will test more extensively later tonight and give the approval if everything all right.

**Review by JucanAndreiDaniel — APPROVED** [review]

**Pull request merged**